Source/Urho3D/UI/FontFace.h

```diff
@@ -75,21 +75,17 @@ class URHO3D_API FontFace : public RefCounted
     virtual bool Load(const unsigned char* fontData, unsigned fontDataSize, float pointSize) = 0;
     /// Return pointer to the glyph structure corresponding to a character. Return null if glyph not found.
     virtual const FontGlyph* GetGlyph(unsigned c);
-
-    /// Return if font face uses mutable glyphs.
-    virtual bool HasMutableGlyphs() const { return false; }
-
     /// Return the kerning for a character and the next character.
     float GetKerning(unsigned c, unsigned d) const;
     /// Return true when one of the texture has a data loss.
     bool IsDataLost() const;
+    /// Return if font face uses mutable glyphs.
+    virtual bool HasMutableGlyphs() const { return false; }
 
     /// Return point size.
     float GetPointSize() const { return pointSize_; }
-
     /// Return row height.
     float GetRowHeight() const { return rowHeight_; }
-
```

Comment:
This is good. Those extra empty lines have no reason to exist :+1:

Reply:
Indeed, they are in the way of those very informative comments